src/testUtils.js

```diff
@@ -152,16 +152,6 @@ export async function destroyReport(report) {
     },
   });
 
-  try {
```

Comment:
Why remove this and not wrap it in a similar "if" statement to the ones below?

Reply:
It was not removed, but moved to after the ActivityReport entries are removed which reference the user table.

Reply:
Should I remove the try/catch's as all the fk issues have now been addressed?

Reply:
I'd say yes, I think if we run into an error that is not a fk error we want to error out so we can see the error